dist/src/main/java/io/camunda/zeebe/shared/management/BackupEndpoint.java

```diff
@@ -76,7 +76,40 @@ public WebEndpointResponse<?> take(final long backupId) {
   }
 
   @ReadOperation
-  public WebEndpointResponse<?> status(@Selector @NonNull final long id) {
+  public WebEndpointResponse<?> listAll() {
+    return listPrefix(BackupApi.WILDCARD);
+  }
+
+  @ReadOperation
+  public WebEndpointResponse<?> query(@Selector final String prefixOrId) {
+    if (prefixOrId.endsWith(BackupApi.WILDCARD)) {
+      return listPrefix(prefixOrId);
+    }
+    final long id;
+    try {
+      id = Long.parseLong(prefixOrId);
+    } catch (final NumberFormatException e) {
+      return new WebEndpointResponse<>(
+          new Error()
+              .message(
+                  "Id '%s' was neither an number nor a prefix ending with '*'"
```

Comment:
🔧 Following our error guidelines https://github.com/camunda/camunda/wiki/Error-Guidelines

```
"Expected given backup ID to be a number or a prefix ending with '*', but got '%s' which was neither of them."
```

Reply:
Done in 2c30d35dfc1009c6bdda6caa3e41aeaf36da1909